docs/notes/previous_versions/v3.2.2.rst

```diff
@@ -0,0 +1,26 @@
+`Version 3.2.2 (latest) <https://fast-dds.docs.eprosima.com/en/v3.2.2/index.html>`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This release includes the following **fixes**:
+
+#. Fix ``related_sample_identity`` handling in RPC for ROS 2 request messages
+#. Fix compilation in Windows using MinGW compiler
+#. Fix a crash error in ``XMLDynamicParser.cpp`` due to ``wstring_builder`` being created as ``nullptr``
+
+This release includes the following **improvements**:
+
+#. Add support for sequence types in ``IDLParser``
+#. Improve efficiency of Discovery Server routines
+#. Always copy ``type_information`` in ``ReaderProxyData`` and ``WriterProxyData`` copy constructors
+#. Add Fast DDS v2.6.10 to supported releases
```

Comment:
We may need to put this into a literal string to make tests pass, but lets wait for the tests first
```suggestion
#. Add Fast DDS ``v2.6.10`` to supported releases
```